lib/redirect.js

```diff
@@ -1,59 +1,73 @@
 // module dependencies
-var parsePath = require("./parsePath")
-	, precompile = require("./precompile")
-	, compile = require("./compile")
-	, sanitize = require("sanitize-arguments");
+var parsePath  = require("./parsePath"),
+    precompile = require("./precompile"),
+    compile    = require("./compile"),
+    sanitize   = require("sanitize-arguments");
 
 // expose redirect plugin
 module.exports = plugin;
+var application;
```

Comment:
Ouch, global state should be avoided. This will introduce unexpected behavior when using more than one app in a single process.